integration-tests/test/setup-docker-compose-network.ts

```diff
@@ -1,7 +0,0 @@
-import { DockerComposeNetwork } from './shared/docker-compose'
```

Comment:
Why did you delete this files? To be honest, I never used them but I'm not sure if other people did. I always run `docker compose` manually

Reply:
I don't think that these should be part of the repo. It introduces a hard coupling between the tests we define here, and the environment that they run on. It also causes the CLI to do a bunch of unexpected stuff unless an env var is set.